website/static/js/share/stats.js

```diff
@@ -92,6 +100,90 @@ function timeGraph (data) {
     });
 }
 
+Stats.sourcesAgg = function(){
+    var sourcesQuery = {'match_all':{}};
+    var sourcesAgg = {'sources': utils.termsFilter('field','_type')};
+    return {'query' : sourcesQuery, 'aggregations': sourcesAgg ,'filters' : {}}
+};
+
+Stats.sourcesByDatesAgg = function(){
+    var dateTemp = new Date(); //get current time
+    dateTemp.setMonth(dateTemp.getMonth() - 3);
+    var threeMonthsAgo = dateTemp.getTime();
+    var dateHistogramQuery = {'match_all':{}};
+    var dateHistogramAgg = {'sourcesByTimes': utils.termsFilter('field','_type')};
+    dateHistogramAgg.sourcesByTimes['aggregations'] = {'articlesOverTime' :
+        utils.dateHistogramFilter('providerUpdatedDateTime',threeMonthsAgo)};
+    return {'query' : dateHistogramQuery, 'aggregations': dateHistogramAgg ,'filters' : {}}
+};
+
+Stats.timeSinceEpochInMsToMMYY = function(timeSinceEpochInMs)
```

Comment:
I think you can also specify the return format of the dates in the aggregation itself, if you'd prefer.